docker/Dockerfile

```
@@ -1,19 +1,18 @@
+# To build this container, go to ESMValCore root folder and execute :
+# docker build -t ${TAG_OF_YOUR_CHOICE} . -f docker/Dockerfile
 FROM continuumio/miniconda3
 
-# update the conda packages
-RUN conda update -y conda pip
+COPY . /esmvaltool
+WORKDIR /esmvaltool
 
-# install development tools
-RUN apt-get update -y && apt-get install -y \
-    build-essential \
-    curl \
-    unzip
+RUN conda update -y conda pip && conda env update --name base --file environment.yml && conda clean --all -y
+RUN pip install . && pip cache purge
+RUN apt install julia -y
+RUN julia esmvaltool/install/Julia/setup.jl
+RUN Rscript esmvaltool/install/R/setup.R
+RUN rm -r /esmvaltool
 
-# install environment packages
-RUN conda install -c conda-forge -c esmvalgroup -c birdhouse esmvaltool
-
-# run tests
-RUN esmvaltool -h
+RUN esmvaltool -v
```

```suggestion
RUN esmvaltool -h
```
`-v` doesn't work with the new command line interface. Could be `esmvaltool version`?